Add alt's & title's

piotrantosz opened the pull request:
Added important alt's & title's

"Website was looking empty, so I add background with sugar kids photo which looks great with "Learning software for children", also I needed to change footer color to white for better look and change inquiry site. Text was floating in air and with no chance, I needed to make separate cards with background opacity on 0.7. I think we could also change font but it's suggestion for future. Image is compressed to 56kB, I couldn't achieve more."

Boquete commented:
cc: @samdroid-apps ?

samdroid-apps commented:
I think I looked at it and it looked fine.

I'm currently afk for a week though.

On Sun, Dec 13, 2015, 8:38 AM Piotr Antosz notifications@github.com wrote:

> cc: @samdroid-apps https://github.com/samdroid-apps ?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/samdroid-apps/www-sugarlabs/pull/6#issuecomment-164195610
> .

piotrantosz commented:
Fixed/added second type grid to use in inquiry etc. Everything works now.

@samdroid-apps squashed commits

Also changed screenshot with pointing. Now it's not manual but guide showing most important things like type's of AP. That's most important things to now when starting.

tradzik commented:
Good job, I will accept your task, but I hope you merge all the commits into one! Thanks

samdroid-apps commented:
What even is this patch now?  There is like everything.

You are allowed to open 100 pull requests if you want.  Please split this up into multipul patches and pull requests!
